cobrand_hackney/static/_timeline.scss

```diff
@@ -31,15 +31,15 @@
     font-weight: 600;
 }
 
-.lbh-details {
+.lbh-timeline {
```

Comment:
Should this perhaps be `.lbh-details, .lbh-timeline` - in case details element is used somewhere else not in a timeline?

Reply:
I changed this so that the summary-list font-size rule would apply to the print-only copies of the details elements too (which _don’t_ have a `.lbh-details` class, but _are_ inside the `.lbh-timeline` parent). But, yeah, no need for _everything_ to be shifted up to `.lbh-timeline`, so I’ve made a fixup that sorts out the font-size in a separate ruleset.